index.html

```diff
@@ -12,7 +12,7 @@
 		<!-- Header -->
 			<header id="header">
 			<header>
-				<div class="logo"><a href="#">Welcome to our awesome site</a></div>
+				<div class="logo"><a href="#">Welcome to our awesome site !!!</a></div>
```

Comment:
```suggestion
				<div class="logo"><a href="#">Welcome to our awesome site !!!</a></div>
``` 
remove the exclamation marks